packages/webpack-cli/lib/groups/resolveAdvanced.js

```diff
@@ -28,6 +32,21 @@ const resolveAdvanced = (args) => {
             finalOptions.options.plugins = [prefetchVal];
         }
     }
+    if (analyze) {
+        if (packageExists('webpack-bundle-analyzer')) {
+            // eslint-disable-next-line node/no-extraneous-require
+            const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
+            const bundleAnalyzerVal = new BundleAnalyzerPlugin();
+            if (finalOptions.options && finalOptions.options.plugins) {
+                finalOptions.options.plugins.unshift(bundleAnalyzerVal);
+            } else {
+                finalOptions.options.plugins = [bundleAnalyzerVal];
+            }
+        } else {
+            error(`It looks like ${yellow('webpack-bundle-analyzer')} is not installed.`);
+            process.exitCode = 2;
+        }
+    }
```

Comment:
Can we ask to install `webpack-bundle-analyzer` here?

Reply:
Yes